fs_xgspon_mod.py

```diff
@@ -743,7 +763,7 @@ def parse_vlan_filter(vf):
 
     parse_persist = s.add_parser("persist")
     parse_persist.add_argument("--onu_ip", default="192.168.100.1")
-    parse_persist.add_argument("isp_ont_serial", type=parse_serial)
+    parse_persist.add_argument("serial", type=parse_serial)
```

Comment:
I would like to see this backed out, just so that the argparse help is slightly clearer for users of ISPs where the distinction between the two matters. Mostly because `install` and `persist` are going to be used at the boundary where the active serial from the stick's perspective changes, so I think the extra clarity is useful.